packages/vats/test/test-boot.js

```diff
@@ -103,13 +103,16 @@ const testRole = (ROLE, governanceActions) => {
       { argv: argvByRole[ROLE], governanceActions },
     );
 
-    const fakeVatAdmin = makeFakeVatAdmin(() => {}).admin;
+    const { admin: fakeVatAdmin } = makeFakeVatAdmin(() => {});
     const fakeBundleCaps = new Map(); // {} -> name
     const getNamedBundleCap = name => {
       const bundleCap = harden({});
       fakeBundleCaps.set(bundleCap, name);
       return bundleCap;
     };
+    const getBundleIDByName = name => `b1-${name}`;
+    const getBundleCap = id => devices.vatAdmin.getBundleCap(id);
```

Comment:
Would it make sense for this to be `const getBundleCap = devices.vatAdmin.getBundleCap`? Or does that obscure the fact that we're defining a function here?